UKHomeOffice/cop-private-workflow-tasklist
# COP-1187 Update form list design

**AC**

Remove all back to dashboard links

- Remove padding from Main.jsx <main>
- Remove back functionality from withShiftCheck.jsx

Update headings and intro text
Update styling of list of form links

- Add GDS task list styling (tasklist.scss)
- Update FormsListPage.jsx to use lists
- Remove FormsListPage.css as no longer needed
- Add a.govuk-link {text-decoration: underline} to ensure links are underlined


**Additional**

- Swap position of Reports and Cases in header to match dashboard order

----

**To Test**

- log into COP
- click on 'forms' from menu or dashboard

> you should see a list of forms in the new styling as per prototype (link in Jira)
> you should see the title and intro text as per prototype
> you should NOT see the left hand column for Filtering
> you should NOT see a back to dashboard link

- click on a form name

> the form should load

- click on 'back to forms' link

> you should return to the forms list
